### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by jasonmccay**
Added in some additional context, additional examples (for SMS), expanded out the instruction set for Rails, and talked some about what Enveloop is, links to the site, etc.

I still want to add in instructions for using `send_raw`, but getting this out for now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,72 @@
...
+```
+
+## Using the Enveloop Gem
 
-Or install it yourself as:
+The Enveloop Gem includes helpful methods to interact with the Enveloop API, including:
 
-    $ gem install enveloop
+* `send_message`
+* `send_raw`
```
Comment (benwyrosdick) [inline README.md]: do we want to advertise this?

Comment (benwyrosdick) [reply README.md]: instead of send raw we might show them how to send HTML without a template

Comment (jasonmccay) [reply README.md]: I didn't do a write up for it -- per our conversation. It's in the code. But, I am indifferent. We can remove it.

Comment (jasonmccay) [reply README.md]: Removed `send_raw`. Also, I did leave in, and expand a bit, the discussion around how to use `send_message` and send HTML.

Comment (benwyrosdick) [reply README.md]: awesome .. thank you

**Pull request merged**